iphone/Classes/TiUIScrollView.m

```diff
@@ -362,7 +362,6 @@ - (void)setContentHeight_:(id)value
 
 - (void)setRefreshControl_:(id)args
 {
-#ifdef USE_TI_UIREFRESHCONTROL
```

Comment:
Why this macro removed?

Reply:
Oh, this was removed by accident because changes from master where pushed in and then i reverted back to a previous state. Fixed!

Reply:
Thanks!